slider/src/test/clusters/emr/README.md

```diff
@@ -1,22 +0,0 @@
-<!--
```

Comment:
Basically rename all the files which were previously some-module/README.md and are now some-module.md back to their original names